### ISSUE TITLE
Copying to clipboard does not work for disabled code interface

### ISSUE THREAD

**Issue by DL1PZ** [directus/directus#11684]
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

When trying to select text from a text field which is disabled, the text is not copied to the clipboard.

### To Reproduce

Open a collection entry for that you don't have writing permissions. Select some text from a field and press CTRL-c. Paste your clipboard to another window.

Same for any text field when opening an old revision of a collection entry.

### What version of Directus are you using?

9.5.1

### What version of Node.js are you using?

17.3.1

### What database are you using?

PostgreSQL 13.5

### What browser are you using?

Firefox, Chrome

### What operating system are you using?

Linux, Windows

### How are you deploying Directus?

npx

**Comment by licitdev** [comment]
Hi @DL1PZ, I am unable to reproduce it based on the information provided. 

May I ask which field are you selecting text from? I have created a text input field with only read permissions, and I am able to copy the value by selecting the text.

Alternatively, you may copy the raw value via the following method:

https://user-images.githubusercontent.com/26413686/154847448-3c214544-5ee9-4c33-a248-0bedfb3dd571.mov

Please kindly provide a screen recording or detailed steps to reproduce the issue. 
Thank you.

**Comment by DL1PZ** [comment]
Thank you very much for trying to reproduce the issue.

The issue does not affect fields which are set readonly. As explained in my initial post, it affects fields for that the current user does not have write permissions as well as fields in the revision history view. I hope that helps. If not, I can provide a screen recording.

The hint to access the raw value helps a lot as a workaround.

Nevertheless, the problem should be solved to provide the expected behavior when selecting text.

**Comment by licitdev** [comment]
@DL1PZ Please provide screenshots or a screen recording. Thank you.

On a side note, I wonder if it is an issue with `Ctrl+C` as I am testing this on MacOS using `Cmd+C`.

### Testing notes

I have tested with the following separately:

Read only permissions where user does not have write permissions.

![Screenshot 2022-02-20 at 10 49 28 PM](https://user-images.githubusercontent.com/26413686/154848461-3b5c9377-3f32-4420-a4e8-81acec46d27f.png)

Readonly field conditions to disable a field even for user with write permissions

![Screenshot 2022-02-20 at 10 50 30 PM](https://user-images.githubusercontent.com/26413686/154848495-d6e33183-2a81-4b93-bcab-f7e9ae27338b.png)

And I was unable to reproduce the issue both when copying text from the field and from the revisions drawer preview. The text could still be selected and copied.

![Screenshot 2022-02-20 at 10 56 40 PM](https://user-images.githubusercontent.com/26413686/154848757-1eb44c7c-886b-428e-acad-e30e192e778f.png)

The two things I've noticed is that text cannot be copied in the revision's diff and for relational fields' displayed value. Which was why I have asked which field are you trying to copy the text from.

Revision's diff, where the text cannot be copied.

![Screenshot 2022-02-20 at 10 58 55 PM](https://user-images.githubusercontent.com/26413686/154848868-33f50e91-39d4-4f3d-8bf9-741690afb871.png)

Relational field's display text cannot be copied. Example below shows an O2M field.

![Screenshot 2022-02-20 at 11 00 12 PM](https://user-images.githubusercontent.com/26413686/154848937-15978546-ed1d-44dd-a3b8-4dab4589cb73.png) ![Screenshot 2022-02-20 at 11 00 42 PM](https://user-images.githubusercontent.com/26413686/154848939-1c65de9c-2d4c-4b86-9042-7bff96b90427.png)

**Comment by DL1PZ** [comment]
During creation of screenshots, I noticed that I was not aware of important details. Sorry for any confusion... 

The problem affects only the **code editor**. Readonly fields are affected the same as fields not having writing permissions, but only for the code editor. The problem is not limited to CTRL-c, any way of copying is affected: CTRL-c, Linux-style copying by selecting, as well as right-klick, which does not even show the option to copy. Tested on Linux/Chrome, Linux/Firefox and Windows/Firefox.

How to reproduce:

1. Create a text field with code editor as interface and set it to readonly. Default settings for everything else.

![Screenshot at 2022-02-20 16-20-13](https://user-images.githubusercontent.com/91094750/154849973-dbcc7b21-0560-46e0-829c-535256103059.png)

![Screenshot at 2022-02-20 16-20-22](https://user-images.githubusercontent.com/91094750/154849985-b55bcbb2-eaaa-4806-a7b5-eceffb460901.png)

![Screenshot at 2022-02-20 16-20-29](https://user-images.githubusercontent.com/91094750/154849988-7f61beef-193e-4ea9-bffd-4196e324a62d.png)

2. Try copying the value from the code editor

![Screenshot at 2022-02-20 16-23-30](https://user-images.githubusercontent.com/91094750/154850055-d2b871b7-f92e-4006-8f91-a68edee55ec4.png)

Thank you very much again!
